src/constants/states.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
The reason this is not an array is so that it gets parsed into a hashtable in javascript and gives O(1) lookup. Basically a set.